testplan/common/report/base.py

```diff
@@ -57,11 +57,17 @@ class Report:
     exception_logger = ExceptionLogger
 
     def __init__(
-        self, name, description=None, uid=None, entries=None, parent_uids=None
+        self,
+        name: str,
+        description: Optional[str] = None,
+        instance_name: Optional[str] = None,
+        uid: Optional[str] = None,
+        entries: Optional[list] = None,
+        parent_uids: Optional[List[str]] = None,
     ):
         self.name = name
         self.description = description
-
+        self.instance_name = instance_name or name
```

Comment:
suggest rename instance_name to definition_name.
And the filters has a match_definition param, is the behavior per its name??

Reply:
tbh `match_definition` is more like `match_uid` to me